.changeset/cold-scissors-hunt.md

```diff
@@ -0,0 +1,6 @@
+---
+'@twilio-paste/design-tokens': major
+'@twilio-paste/theme': major
+---
+
+[Design Tokens, Theme] Deprecate the legacy console theme in favor of the default theme.
```

Comment:
Do I need 2 separate changesets? Also, is this an adequate message or do I need to add more information?